### DISCUSSION TITLE
Unable to edit Role in TE3

### DISCUSSION THREAD

**Discussion by mvdvook** [TabularEditor/TabularEditor3#593] [🙋♀️TE3 Q&A] [answered]
Hi,

In TE 2 I can edit my role definitions, in TE 3 I can see them but not edit them. 
Is that correct?

Reinstalled, restarted etc did not help.

**Comment by otykier** [comment]
TE3 should let you edit roles just like TE2. Could you share a screenshot or clarify what exactly you are trying to edit?

**Comment by mvdvook** [comment]
Hi,

I can see the roles, and click on them, and see the code, but I cannot type anything in the code. The keyboard does not work.
WIth measures, it works brilliantly.
![ScreenshotRoles](https://user-images.githubusercontent.com/42534259/179568836-1a36a150-6d20-4a88-b7a7-f91a71f3d567.png)
Build 3.3.8214.

I recently updated, did not help

![Screenshot Version TE3](https://user-images.githubusercontent.com/42534259/179569230-a44dbda9-9e85-420e-8bd0-9c017d0b3602.png)

Again, I do exactly the same in TE2, no problems. 
If you need any more info ( of a live demo) let me know :)

Maaike

**Reply by otykier** [reply, marked as answer]
Is this when connecting to a Power BI Desktop (pbix) model? In that case, you may have to enable the **Allow unsupported modeling operations** option under **Tools > Preferences > Tabular Editor**.

**Reply by mvdvook** [reply]
Thanx Daniel! That's it indeed. Works perfectly now :)
